src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/FeePaymentOutcomeHWFCallBackHandler.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.civil.handler.callback.user;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableMap;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.Callback;
+import uk.gov.hmcts.reform.civil.callback.CallbackHandler;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.model.BusinessProcess;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.citizen.HWFFeePaymentOutcomeService;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_START;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.SUBMITTED;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class FeePaymentOutcomeHWFCallBackHandler extends CallbackHandler {
+
+    private final ObjectMapper objectMapper;
+    private final HWFFeePaymentOutcomeService hwfFeePaymentOutcomeService;
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return new ImmutableMap.Builder<String, Callback>()
+            .put(callbackKey(ABOUT_TO_START), this::emptyCallbackResponse)
+            .put(callbackKey(ABOUT_TO_SUBMIT), this::submitFeePaymentOutcome)
+            .put(callbackKey(SUBMITTED), this::emptySubmittedCallbackResponse)
+            .build();
+
+    }
+
+    private CallbackResponse submitFeePaymentOutcome(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        CaseData.CaseDataBuilder<?, ?> caseDataBuilder = caseData.toBuilder();
+
+        if (caseData.isHWFTypeClaimIssued()) {
+            LocalDate issueDate = LocalDate.now();
+            caseDataBuilder.issueDate(issueDate).build();
+            caseDataBuilder.businessProcess(BusinessProcess.ready(CaseEvent.CREATE_CLAIM_SPEC_AFTER_PAYMENT));
```

Comment:
just want to verfiy  , in case no remission

Reply:
For No Remission case too.. claimant has to pay the claim fee offline and then the caseworker will submit the 'Fee Payment Outcome' event